Marlin/src/lcd/extui/anycubic_vyper/dgus_tft.cpp

```diff
@@ -804,6 +787,11 @@ namespace Anycubic {
     changePageOfTFT(page_index, true);
   }
 
+  void DgusTFT::changePageOfTFTToAbout() {
+    sendTxtToTFT_P(getFirmwareName_str(), TXT_VERSION);
```

Comment:
If this and other strings only need to be set once then they can be moved to `DgusTFT::startup`.

Reply:
You are right there is no need to send it every time.
I have try to put 
```C
sendTxtToTFT_P(getFirmwareName_str(), TXT_VERSION);
```
in  `DgusTFT::startup()` but it is not working (it work only on reset not on cold start from poweroff) maybe the screen is not ready to accept data.

Do you have an other place to put it ?

Reply:
It's not of massive importance since it's not a bottleneck or bloating the build, but it's something to consider for the future with displays like this. At this time it's fine, and probably best, to just keep these lines together.